src/index.js

```diff
@@ -38,7 +40,17 @@ module.exports = function({ template, transformFromAst, types }) {
           return;
         }
 
-        const { code } = transformFromAst(types.Program([path.node]));
+        // get the test case body
+        let body = path.node.expression.arguments[1].body;
+
+        // if it's an expression, e.g: () => (expression)
+        if (types.isCallExpression(body)) {
```

Comment:
I wonder if this would make more sense to be:
```
if (!types.isBlockStatement(body)) { 
...
}
```

Reply:
I chose to do it this way because:

1) If I can assume it's a call expression then I know I can create a `return` statement (see further down).
2) I figured the possibilities were mutually exclusive - it's either a block statement or a call expression.